doc/daikon.texinfo

```diff
@@ -2247,10 +2258,11 @@ Static variables of a class have names of the form
 @cindex pre-state variables
 @code{orig(x)} refers to the value of variable @code{x} upon
 entry to a procedure (because the procedure body might modify the value
-of @code{x}).  These variables appear only at @code{:::EXIT} program
-points.  Typically, @code{orig()} variables do not appear in the trace,
-but are automatically created by Daikon when it matches up
-@code{:::ENTER} and @code{:::EXIT@var{nn}} program points.
+of @code{x}).  These variables appear only at @code{:::EXIT} and
```

Comment:
Please don't refill paragraphs (that is, don't remove line breaks).  It makes the diffs bigger and harder to read.